### PR TITLE
Create 2021.35 release notes

### PR DESCRIPTION
**Pull request by latechwriter**
This week's release notes are ready to review/ship. Thanks.

### CODE REVIEW

File: content/release-notes/2021-09-20.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+### Crunch web application
+
+- Fixed an issue that occurred when a tab on a dashboard tile was not allowing the user to change the selection.
+- Improved the behavior in the 'Dashboards Group' panel in situations when one or more of the filter variables have been deleted from the dataset.
+- Fixed an issue that occurred when selecting a category in the 'Dashboard Group' panel that sometimes did not reflect in the rendering of the dashboard.
+- Fixed a bug in [Confirmit integrations](https://help.crunch.io/hc/en-us/articles/4403966631309-How-your-survey-metadata-maps-from-Confirmit-into-Crunch) when response data was smaller than expected.
+- Corrected an issue that occurred when a dashboard tile no longer reflects the column selected for hypothesis testing comparisons after changing the selection in the tab pulldown within that tile.
+- Added a new option in the 'Dashboard Edit Groups' panel for removing a filter that corresponds to a variable that has been deleted from the dataset.
+- Added the ability to download a Crunch Automaton script from the UI when creating a folder structure in a dataset and moving variables into it (a link appears allowing you to download the script).
```
Comment (malecki) [inline content/release-notes/2021-09-20.md:17]:
```suggestion
- Added the ability to download a Crunch Automaton script (a series of `organize` commands) reflecting the current variable folder structure from the ‘Edit variables’ interface.
```

File: content/release-notes/2021-09-20.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
++++
+
+### Crunch web application
+
+- Fixed an issue that occurred when a tab on a dashboard tile was not allowing the user to change the selection.
+- Improved the behavior in the 'Dashboards Group' panel in situations when one or more of the filter variables have been deleted from the dataset.
+- Fixed an issue that occurred when selecting a category in the 'Dashboard Group' panel that sometimes did not reflect in the rendering of the dashboard.
+- Fixed a bug in [Confirmit integrations](https://help.crunch.io/hc/en-us/articles/4403966631309-How-your-survey-metadata-maps-from-Confirmit-into-Crunch) when response data was smaller than expected.
+- Corrected an issue that occurred when a dashboard tile no longer reflects the column selected for hypothesis testing comparisons after changing the selection in the tab pulldown within that tile.
+- Added a new option in the 'Dashboard Edit Groups' panel for removing a filter that corresponds to a variable that has been deleted from the dataset.
```
Comment (malecki) [inline content/release-notes/2021-09-20.md:16]:
```suggestion
- Added the ability to remove a stale reference in dashboard ‘groups’ when a variable has been deleted from the dataset.
```

**Review by malecki — APPROVED** [review]

File: content/release-notes/2021-09-20.md (hunk 1 above)
Comment (latechwriter) [reply content/release-notes/2021-09-20.md:17]: Updated.

**Comment by latechwriter** [comment]
Ready to ship!

**Pull request merged**